desktop/apps/consignments2/client/validate.js

```diff
@@ -0,0 +1,43 @@
+export const numberWarning = value => value && isNaN(Number(value)) && 'Must be a number'
+
+export const onFail = (errors, dispatch) => {
+  const scrollBuffer = 5
+  const firstError = Object.keys(errors)[0]
+  const firstErrorElement = document.getElementsByName(firstError)[0]
+  if (firstErrorElement) {
+    window.scrollTo(0, firstErrorElement.offsetTop - scrollBuffer)
+    firstErrorElement.querySelector('input').focus()
+  }
+}
```

Comment:
Since this function basically only has to do with scrolling to the element, i would make it more descriptive -- `onFail: scrollToError`.